paddle/cinn/pybind/schedule.cc

```diff
@@ -35,13 +35,18 @@ void BindSchedule(py::module *m) {
            py::arg("debug_flag") = false,
            py::arg("err_msg_level") = utils::ErrorMessageLevel::kGeneral,
            py::arg("is_dynamic_shape") = false)
-      .def_static(
-          "make",
-          [](ir::LoweredFunc &ir_func) {
-            ir::ModuleExpr *module_expr = new ir::ModuleExpr({ir_func->body});
-            auto scheduler = std::make_unique<ir::IRSchedule>(*module_expr);
-            return scheduler;
-          })
+      .def_static("make",
+                  [](ir::LoweredFunc &ir_func) {
+                    ir::ModuleExpr *module_expr =
+                        new ir::ModuleExpr({ir_func->body});
+                    auto scheduler = std::make_unique<ir::IRSchedule>(
+                        *module_expr,
+                        -1,
```

Comment:
Usually, we add some parameter comment to -1, true, false input to increase readability.

For example:
/* param_option = */ -1,

Reply:
Ok, I will fix it in next PR.

Reply:
The PR is [#60342](https://github.com/PaddlePaddle/Paddle/pull/60342)